### PR TITLE
M-32: docs(readme): update wording

### PR DESCRIPTION
**Pull request by almereyda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,13 +14,16 @@ cd o-platform
 # Run
 cd shell
 npm run dev
+```
+
+Build & Run with local self-signed Certificate:
 
-#Build & Run with local self-signed Certificate:
```
Comment (jaensen) [inline readme.md:17]: 'Certificate' should be lowercase

**Review by jaensen — COMMENTED** [review]
Nice. But.

File: readme.md [hunk 2]
```diff
@@ -13,14 +14,16 @@ cd o-platform
 # Run
 cd shell
 npm run dev
+```
+
+Build & Run with local self-signed Certificate:
```
Comment (almereyda) [inline readme.md:19]:
```suggestion
Build & Run with local self-signed certificate:
```

**Comment by almereyda** [comment]
Changes were resolved.

Re-review re-requested.

**Comment by almereyda** [comment]
Is this ready to be merged?

**Review by codeho — APPROVED** [review]

**Pull request merged**